backend/game/src/gameplay/score.rs

```diff
@@ -77,6 +83,20 @@ impl Score {
             // Check if player has the custodians
             player_points += i8::from(self.custodians.as_ref() == Some(player));
 
+            // Check if the player has the Shard of the Throne relic
+            if let Some(p) = self.shard_of_the_throne.as_ref() {
```

Comment:
I think this would work if you care about saving 4 LOCs :p
```rust
if Some(player) == self.shard_of_the_throne.as_ref() {
```

Reply:
But my contributions :cry:

No but, very nice 👍🏿